tests/test_lightgbm_converter.py

```diff
@@ -280,10 +280,7 @@ def test_lightgbm_onnx(self):
         onnx_model = hummingbird.ml.convert(model, "onnx", X)
 
         # Get the predictions for the ONNX-ML model
-        session = ort.InferenceSession(onnx_model.SerializeToString())
-        output_names = [session.get_outputs()[i].name for i in range(len(session.get_outputs()))]
-        inputs = {session.get_inputs()[0].name: X}
-        onnx_pred = session.run(output_names, inputs)
+        onnx_pred = onnx_model.predict(X)
```

Comment:
ok, i understand now about the breaking change.  i think this is great!  (well, not the breaking part, but much more usable.)

Reply:
Yea that is what I tought too!